rosdep/python.yaml

```diff
@@ -9521,6 +9521,23 @@ python3-squaternion-pip:
   ubuntu:
     pip:
       packages: [squaternion]
+python3-sshtunnel:
+  debian: [python3-sshtunnel]
+  fedora: [python3-sshtunnel]
+  ubuntu:
+    '*': [python3-sshtunnel]
+    focal:
+      pip:
+        packages: [sshtunnel]
+    jammy:
+      pip:
+        packages: [sshtunnel]
+    kinetic:
```

Comment:
You can remove kineitc and lunar, those aren't Ubuntu OS which ROS 2 releases binaries to (or even has a distribution on)

Reply:
Done: https://github.com/ros/rosdistro/pull/36071/commits/1962b88f2000d984a9bfa48a04670c076482c630